### PR TITLE
Tweak: Add a hook to get manifest data in import CLI command [ED-7671]

### PR DESCRIPTION
**Pull request by udidol**
## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [X] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

* Tweak: Add a hook to get manifest data in Kit import CLI command

### CODE REVIEW

File: core/app/modules/import-export/wp-cli.php [hunk 1]
```diff
@@ -155,6 +155,17 @@ public function import( array $args, array $assoc_args ) {
...
+			/**
+			 * Import Export Manifest Data
+			 *
+			 * Allows 3rd parties to read and edit the kit's manifest before it is used.
+			 *
+			 * @since 3.8.0
+			 *
+			 * @param array $manifest_data The Kit's Manifest data
+			 */
+			apply_filters( 'elementor/import-export/manifest_data', $manifest_data );
```
Comment (ManorHazaz) [inline core/app/modules/import-export/wp-cli.php:167]:
```suggestion
			apply_filters( 'elementor/import-export/wp-cli/manifest_data', $manifest_data );
```

Maybe you should be more specific that the filter is available only in the CLI?

**Review by ghost — CHANGES_REQUESTED** [review]

File: core/app/modules/import-export/wp-cli.php (hunk 1 above)
Comment (udidol) [reply core/app/modules/import-export/wp-cli.php]: I agree. Done.

**Review by ghost — DISMISSED** [review]

**Review by nivnoimanelementor — DISMISSED** [review]
Usually a filter is used to return data, so in this case I would use an action, but other than that it seems fine to me. If it's okay by you, then great.

File: core/app/modules/import-export/wp-cli.php [hunk 2]
```diff
@@ -155,6 +155,17 @@ public function import( array $args, array $assoc_args ) {
...
+			/**
+			 * Import Export Manifest Data
+			 *
+			 * Allows 3rd parties to read and edit the kit's manifest before it is used.
+			 *
+			 * @since 3.8.0
+			 *
+			 * @param array $manifest_data The Kit's Manifest data
+			 */
+			apply_filters( 'elementor/import-export/wp-cli/manifest_data', $manifest_data );
```
Comment (udidol) [inline core/app/modules/import-export/wp-cli.php:167]:
```suggestion
			$manifest_data = apply_filters( 'elementor/import-export/wp-cli/manifest_data', $manifest_data );
```

**Comment by udidol** [comment]
> Usually a filter is used to return data, so in this case I would use an action, but other than that it seems fine to me. If it's okay by you, then great.

You are right - I missed it. I added an overwrite of the manifest variable returning from the filter. good call @nivnoimanelementor

**Review by yossi-elementor — APPROVED** [review]

**Pull request merged**